jsonschema-validation.xml

```diff
@@ -367,6 +367,17 @@
                 </t>
             </section>
 
+            <section title="contains">
+                <t>
+                    The value of this keyword MUST be an object.  This object MUST be
+                    a valid JSON Schema.
+                </t>
+                <t>
+                    An array instance is valid against "contains" if at least one of
+                    its elements is valid against the given schema.
+                </t>
```

Comment:
No, an empty array fails that schema, because an empty array passes the schema `"items": {"not": {...}}`, so the outer `not` causes it to fail overall.